src/resources/postcss/tribe-events-single-blocks.pcss

```diff
@@ -15,16 +15,14 @@
 ============================================= */
 .tribe-block__event-website {
 	font-family: var(--tec-font-family-sans-serif);
-	padding: 20px 0;
 
 	a {
 		background-color: var(--tec-color-accent-primary);
 		border: none;
 		border-radius: 4px;
 		font-size: var(--tec-font-size-3);
 		font-weight: var(--tec-font-weight-bold);
-		min-height: 40px;
-		padding: 0 20px;
+		padding: 10px 20px;
```

Comment:
can we use spacer props here?
either `var( --tec-spacer-1 ) var(--tec-spacer-4)` (8px 20px)
or `var( --tec-spacer-2 ) var(--tec-spacer-4)` (12px 20px)

_Always be using custom props for margin and padding whenever possible_ 😉

Reply:
Yes yes, I've updated this.